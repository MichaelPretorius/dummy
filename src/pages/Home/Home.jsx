import React from 'react';

import { Header } from '../../styles';

export const Home = () => {
  return (
    <Header center secondary>
      Home
    </Header>
  );
};

export default Home;
