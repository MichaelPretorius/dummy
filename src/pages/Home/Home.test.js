import React from 'react';
import { shallow } from 'enzyme';

import { Home } from './Home';

describe('Home component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Home />);
  });

  test('should render Home component', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
