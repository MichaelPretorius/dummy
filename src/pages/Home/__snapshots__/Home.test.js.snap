// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render Home component 1`] = `
"<Header center={true} secondary={true}>
  Home
</Header>"
`;
